docs/ops/deployment/kubernetes.md

```diff
@@ -112,20 +153,38 @@ spec:
       containers:
       - name: jobmanager
         image: flink:latest
-        args:
-        - jobmanager
+        workingDir: /opt/flink
+        command: ["/bin/bash", "-c", "$FLINK_HOME/bin/jobmanager.sh start;\
+          while :;
+          do
+            if [[ -f $(find log -name '*jobmanager*.log' -print -quit) ]];
+              then tail -f -n +1 log/*jobmanager*.log;
+            fi;
+          done"]
```

Comment:
Could this be part of the `entrypoint.sh`?

Reply:
IMO, there should be another issue to discuss about modifying the `entrypoint.sh`. Even I include this part as `entrypoint.sh`, since I already use `command` to start the container, the default `entrypoint.sh` would be useless for me.

Reply:
Of course, one would have to use the entrypoint.sh then and not specify `command`. But I'm ok with discussing this as a separate issue and keeping it out of this PR's scope.